ServerStatisticsPublisher/TopicPublisher.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Text;
+using Newtonsoft.Json;
+using RabbitMQ.Client;
+using Configurations;
+using ServerStatisticsService;
+
+namespace ServerStatisticsPublisher
+{
+    public static class TopicPublisher
+    {
+        public static async Task Publish(IModel channel)
+        {
+            channel.ExchangeDeclare("topic-ServerStatistics-exchange" , ExchangeType.Topic);
```

Comment:
these topics and other hardcoded strings are used in multiple projects, would be better to have them in one class file as constants